### DISCUSSION TITLE
Have you considered a "Hobby" tier?

### DISCUSSION THREAD

**Discussion by Wobbley** [supabase/supabase#10033] [Questions] [answered]
**TL;DR Have you considered a "Hobby" tier? Which does not pause and the projects does not count towards the free quota?**

Hey

I have tried building some of my recent projects with Supabase instead of Firebase, and overall I am very happy with the experience with on caveat: The free tier is too restrictive.

Don't get me wrong, I understand that you need to make money, and I have no issue with paying for it, as long as I feel that I get value. My current issue is that the two current tiers are either "Free" or "Full Scale Production". Most of my projects are smaller projects used by my family or friends, and shelling out 25$ per project is not sustainable

I would gladly pay for a "Hobby" tier, where it does not count towards the free quota and does not pause. It could have the same limitations as free.

**Comment by Wobbley** [comment, marked as answer]
I found a similar question [here](https://github.com/supabase/supabase/discussions/8243), my initial searches did not get any hits, sorry for making a duplicate. You can close this
